go/paddlecloud/restclient.go

```diff
@@ -8,22 +8,31 @@ import (
 	"io/ioutil"
 	"mime/multipart"
 	"net/http"
+	"net/url"
 	"os"
 )
 
-// HTTPOK is ok status of http api call
+// HTTPOK is ok status of http api call.
 const HTTPOK = "200 OK"
 
-var httpTransport = &http.Transport{}
+type RestClient struct {
```

Comment:
A struct with only one member seems not necessary. If we want to implement multi-threading download processes, we can use a more simple function like:
```go
func GetChunk(targetURL string, query url.Values, nThreads int) (*http.Response, error) {
    req, err := makeRequest()
    // use a thread pool of http.client to do multi-threading
}
```

Then, we can call this functions by:
```go
resp := restclient.GetChunk()
```
instead of

```go
client := NewRestClient()
resp := client.GetChunk()
```